hbase-asyncfs/pom.xml

```diff
@@ -194,6 +199,26 @@
           <scope>test</scope>
         </dependency>
       </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <!--
+				We do not need this hack when building with hadoop 2.x, so exclude it here.
```

Comment:
This is a bit misleading. The reason for the exclusion is that we cannot get it to work otherwise.